src/test/java/seedu/address/testutil/TypicalClients.java

```diff
@@ -88,6 +88,13 @@ public static TbmManager getTypicalTbmManager() {
     }
 
     public static List<Client> getTypicalClients() {
-        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
+        Client alice = new ClientBuilder(TypicalClients.ALICE).build();
+        Client benson = new ClientBuilder(TypicalClients.BENSON).build();
+        Client carl = new ClientBuilder(TypicalClients.CARL).build();
+        Client daniel = new ClientBuilder(TypicalClients.DANIEL).build();
+        Client elle = new ClientBuilder(TypicalClients.ELLE).build();
+        Client fiona = new ClientBuilder(TypicalClients.FIONA).build();
+        Client george = new ClientBuilder(TypicalClients.GEORGE).build();
+        return new ArrayList<>(Arrays.asList(alice, benson, carl, daniel, elle, fiona, george));
```

Comment:
Can remove TypicalClients.
Also, no need for `new ArrayList` since Arrays.asList already returns a list

Reply:
Resolved